documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -76,217 +73,135 @@ controlPlane:
     replicas: 3
 ----
 |
-|Replicas sets the number of control plane (master) nodes that are to be included as part of your {product-title} cluster.
-
+|Replicas sets the number of control plane (master) nodes included as part of the {product-title} cluster.
 
 ifeval::[{release} >= 4.4]
+
+ifeval::[{release} <= 4.6]
 a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the
 provisioning network. ({product-title} 4.4 only)
 endif::[]
 
-| `hosts`
-|
-| Details about bare metal hosts to use to build the cluster.
+endif::[]
 
 
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
 
 | [[apivip]]`apiVIP` | `api.<clustername.clusterdomain>` | The VIP to use for internal API communication.
 
+This setting must either be provided or pre-configured in the DNS so that the
+default name resolves correctly.
 
-This setting must either be provided or pre-configured in DNS so that the
-default name resolve correctly.
-
-| `disableCertificateVerification` | `False` | This parameter is needed when we use `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses.
+| `disableCertificateVerification` | `False` | `redfish` and `redfish-virtualmedia` need this parameter to manage BMC addresses. The value should be `True` when using a self-signed certificate for BMC addresses.
 
 | [[ingressvip]]`ingressVIP` | `test.apps.<clustername.clusterdomain>` | The VIP to use for ingress traffic.
 
 ifeval::[{release} < 4.5]
-This setting must either be provided or pre-configured in DNS so that the
-default name resolve correctly.
+Provide this setting or pre-configure it in the DNS so that the default name resolves correctly.
 |[[dnsVIP]]`dnsVIP` | | The VIP to use for internal DNS communication.
 
 This setting has no default and must always be provided.
 endif::[]
+
 |===
 
+
 [cols="1,1,3", options="header"]
 .Optional Parameters
 |===
 |Parameters
 |Default
 |Description
 
+
 ifeval::[{release} > 4.3]
 |`provisioningDHCPExternal`
 | false
-|Defines if external DHCP will be used or the one configured by installer
+|Defines if the installer uses an external DHCP or the provisioner node DHCP.
 
 |`provisioningDHCPRange`
-| `172.22.0.10,172.22.0.100`
-|Defines the IP range to use for hosts on the provisioning network.
+|`172.22.0.10,172.22.0.100`
+|Defines the IP range for nodes on the `provisioning` network.
 
-a| [[provisioningNetworkCIDR]] `provisioningNetworkCIDR`
-| `172.22.0.0/24`
-| The CIDR for the network to use for provisioning. This option is required when using IPv6 addressing on the provision network.
+a|`provisioningNetworkCIDR`
+|`172.22.0.0/24`
+|The CIDR for the network to use for provisioning. This option is required when using IPv6 addressing on the `provisioning` network.
```

Comment:
I think we should change this to

This option is required when not using the default 172.22.0.0/24 address range.